source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
                  -H "Content-Type: application/json" \
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
-                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                   "endpoint_type": "general",
+                   "comment": "<comment-about-endpoint>"
                  }'
 
-         .. step:: Modify Your VPC Private DNS
 
-            Use the AWS CLI to modify your VPC private DNS settings. This
-            ensures that your App can resolve the private endpoint's DNS name
-            to the correct IP address.
+Add a Sync Endpoint
+~~~~~~~~~~~~~~~~~~~
+
+Sync Endpoints require that you also have a General Endpoint associated with
```

Comment:
I'm not quite sure I understand what this is trying to communicate. I wonder if we can reword this to be a little more explicit? If I'm interpreting this correctly, maybe something like:

> Every Sync Endpoint must be paired with a General Endpoint created using the same generated command.